HUD/N_0xf83d0febe75e62c9.md

```diff
@@ -5,20 +5,26 @@ ns: HUD
 
 ```c
 // 0xF83D0FEBE75E62C9
-void _0xF83D0FEBE75E62C9(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7);
+void _0xF83D0FEBE75E62C9(Blip blip, Any p1, Any p2, float width, Any p4, float length, float heading, Any p7);
 ```
 
 ```
-NativeDB Added Parameter 9: Any p8
+NativeDB Added Parameter 9: int hudColour
 NativeDB Introduced: v1290
 ```
 
+Sets the 'data' for a cone for a blip
+
+See [here](https://docs.fivem.net/docs/game-references/hud-colors/) for the hud colours for the cone
+
+Name in scripts: `SETUP_FAKE_CONE_DATA()`
+
 ## Parameters
-* **p0**:
-* **p1**:
-* **p2**:
-* **p3**:
-* **p4**:
-* **p5**:
-* **p6**:
-* **p7**:
+* **blip**: A blip from an entity, for example ch_prop_ch_cctv_cam_01a
+* **p1**: -1.0 in decompiled scripts (unk)
+* **p2**: 1.0 in decompiled scripts (unk)
+* **width**: Width of the cone (0.0 to ~3.0) 
+* **p4**: 1.0 in decompiled scripts (unk)
+* **length**: Length of the cone. (8.2 is 'normal' sized
+* **heading**: Not your typical heading. Can be calculated with (heading + 180.00) * 0.017453292
```

Comment:
That `0.017453292` seems to be `π/180` as in Pi divided by 180, a.k.a. `3.14159265359/180`

Reply:
Rotation in radians, huh? This is (for me) the easiest approach and doesn’t over complicate it. If needed ill edit it _again_ and commit it

Reply:
> Rotation in radians, huh? This is (for me) the easiest approach and doesn’t over complicate it. If needed ill edit it _again_ and commit it

Yeah, now I recall that the game itself does this for other native methods as in degree to radian conversions or so. Anyhow, this is good, thanks!